_source/api/logzio-public-api.yml

```diff
@@ -640,6 +645,146 @@ paths:
           schema:
             $ref: '#/definitions/ScrollResponse'
 
+  # ::::: Archive
+  /archive/v2/settings:
+    get:
+      summary: Retrieve archiving settings
+      description: Gets the current archiving settings for a time-based log account.
+      tags:
+        - Archive logs
+      operationId: getSettingsForAccount
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            $ref: '#/definitions/ArchiveSettingsResponse'
```

Comment:
@shalper where is `#/definitions/ArchiveSettingsResponse` defined?

Reply:
@idohalevi The definition begins on line 3315.